components/BlurhashImage.vue

```diff
@@ -38,6 +49,11 @@ export default Vue.extend({
       type: Object as () => BaseItemDto,
       required: true
     },
+    imageType: {
+      type: String as () => ImageType.Primary,
```

Comment:
This isn't right. The type isn't ImageType.Primary, but ImageType (with a default value of ImageType.Primary.